### ISSUE TITLE
Support django default styling of paginator 

### ISSUE THREAD

**Issue by bokchan** [shinneider/django-admin-inline-paginator#9]
I integrated `django-admin-inline-paginator` into our project and it works great!  Thanks a lot! 

However the package defines custom css classes in the [paginator.html](https://github.com/shinneider/django-admin-inline-paginator/blob/master/django_admin_inline_paginator/templates/admin/tabular_paginator.html) template 

It means that the default django styling of the paginator is overridden. 

We experienced that using it together with  the default django flat theme resulted in the available page links being rendered as white text on white background. 

I suggest a solution where the pagination style is generated with the same classes as in https://github.com/django/django/blob/main/django/contrib/admin/templates/admin/pagination.html

**Comment by bokchan** [comment]
I created a branch with the suggested solution and a few other improvements, https://github.com/bokchan/django-admin-inline-paginator/tree/support-django-default-styling

- Support hiding of next/prev links
- Use default django paginator styles in tabular_paginator template
- Update failing unittests

**Comment by shinneider** [comment]
@bokchan open a PR to my repo...
is a great fixes and options... 
if you open this weekend, i evaluate merge and release a new version with your fixes on Monday

**Comment by bokchan** [comment]
@shinneider  I won't have time this weekend but I will try to make it happen next week.

**Comment by paul-nameless** [comment]
Hello guys, just faced the same issue.
It would be very helpful if you'll fix it!

**Comment by shinneider** [comment]
@bokchan  make sense

A generic solution is a best for this case. but i no have much time for develop it.

we keep this pr open, in case someone with availability appears.

**Comment by bokchan** [comment]
Makes sense @shinneider . I am also quite busy but I will try to work on the issue if I get some extra time.
